lib/src/main/kotlin/ca/aqtech/mctreesearch4j/MDP.kt

```diff
@@ -1,12 +1,9 @@
 package ca.aqtech.mctreesearch4j
-abstract class MDP<TState, TAction> {
-    abstract fun transition(state: TState, action: TAction) : TState
-
-    abstract fun reward(previousState: TState?, action: TAction?, state: TState) : Double
-
-    abstract fun initialState() : TState
 
-    abstract fun isTerminal(state: TState) : Boolean
-
-    abstract fun actions(state: TState) : Collection<TAction>
+abstract class MDP<TState, TAction> {
+    abstract fun transition(state: TState, action: TAction): TState
```

Comment:
Some minor style changes since apparently for functions, there's shouldn't be a space between paranthesis and colon i.e. not `function() : returnType` but instead `function(): returnType`